hack/olm-registry/olm-artifacts-template.yaml

```
@@ -1,5 +1,7 @@
 apiVersion: v1
 kind: Template
+metadata: selectorsyncset-template
```

I think this should still be `.metadata.name` right?

```suggestion
metadata:
  name: selectorsyncset-template
```